apps/asap-server/src/controllers/users.ts

```diff
@@ -284,6 +290,28 @@ export default class Users {
     return parseGraphQLUser(items[0]);
   }
 
+  async updateAvatar(id: string, avatar: Buffer): Promise<UserResponse> {
+    const form = new FormData();
+    form.append('file', avatar, {
+      filename: `${id}.jpg`,
+      contentType: 'image/jpeg',
```

Comment:
Are we enforcing only jpeg? I have allowed image/* in the frontend

Reply:
Can you pass the content type as an header? I don't want to be looking into the buffer to figure that out 😅

Reply:
As discussed on zoom: the backend will implement a dataURI parser and extract the type from that.